backend/test/service/GooglePlacesService.test.ts

```diff
@@ -0,0 +1,53 @@
+import { AxiosError } from 'axios';
+import { GooglePlacesService } from '../../src/service/GooglePlacesService';
+import * as dotenv from 'dotenv';
+
+/* eslint-disable  @typescript-eslint/no-unused-vars */
+// todo: remove eslint-disable once we add assertions
+describe('Unit tests for GooglePlacesService', function () {
+  const googlePlacesAPI = new GooglePlacesService();
+
+  beforeEach(function () {
+    dotenv.config();
+  });
+
+  test('Search for a place using restaurant name, should be successful', () => {
+    return googlePlacesAPI
+      .getGooglePlacesSearch('Jinya')
+      .then((result: JSON) => {
+        // todo: need some assertions
```

Comment:
I had to change this from `AxiosResponse<JSON>` to `JSON` because the compiler was giving me an error. Also had to change `dotenv` to an import rather than a require.

I made the `test` folder structure the same as the `src` folder structure, so the unit test for `GooglePlacesService` would be in the same `service` folder inside `test`. It's something I find kind of useful, I can change it back if other people don't want it to be like that.